dev_suites/demo_ig_stu1/groups/demo_group.rb

```diff
@@ -28,6 +28,7 @@ class DemoGroup < Inferno::TestGroup
     fhir_client :this_client_name do # name optional
       url :url # if symbol just use a input?
       headers Authorize: 'something'
+      token :bearer_token
```

Comment:
I could be convinced otherwise, but I think we should be explicit and call this `bearer_token` rather than just `token`.

Reply:
Makes sense. Updated `token` to `bearer_token`